shared/src/metabase/shared/util/currency.cljc

```diff
@@ -0,0 +1,851 @@
+          :rounding 0,
+          :code "SGD",
+          :name_plural "Singapore dollars"}],
+   [:SOS {:symbol "Ssh",
+          :name "Somali Shilling",
+          :symbol_native "Ssh",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "SOS",
+          :name_plural "Somali shillings"}],
+   [:SYP {:symbol "SY£",
+          :name "Syrian Pound",
+          :symbol_native "ل.س.‏",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "SYP",
+          :name_plural "Syrian pounds"}],
+   [:THB {:symbol "฿",
+          :name "Thai Baht",
+          :symbol_native "฿",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "THB",
+          :name_plural "Thai baht"}],
+   [:TND {:symbol "DT",
+          :name "Tunisian Dinar",
+          :symbol_native "د.ت.‏",
+          :decimal_digits 3,
+          :rounding 0,
+          :code "TND",
+          :name_plural "Tunisian dinars"}],
+   [:TOP {:symbol "T$",
+          :name "Tongan Paʻanga",
+          :symbol_native "T$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TOP",
+          :name_plural "Tongan paʻanga"}],
+   [:TRY {:symbol "TL",
+          :name "Turkish Lira",
+          :symbol_native "TL",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TRY",
+          :name_plural "Turkish Lira"}],
+   [:TTD {:symbol "TT$",
+          :name "Trinidad and Tobago Dollar",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TTD",
+          :name_plural "Trinidad and Tobago dollars"}],
+   [:TWD {:symbol "NT$",
+          :name "New Taiwan Dollar",
+          :symbol_native "NT$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "TWD",
+          :name_plural "New Taiwan dollars"}],
+   [:TZS {:symbol "TSh",
+          :name "Tanzanian Shilling",
+          :symbol_native "TSh",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "TZS",
+          :name_plural "Tanzanian shillings"}],
+   [:UAH {:symbol "₴",
+          :name "Ukrainian Hryvnia",
+          :symbol_native "₴",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "UAH",
+          :name_plural "Ukrainian hryvnias"}],
+   [:UGX {:symbol "USh",
+          :name "Ugandan Shilling",
+          :symbol_native "USh",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "UGX",
+          :name_plural "Ugandan shillings"}],
+   [:UYU {:symbol "$U",
+          :name "Uruguayan Peso",
+          :symbol_native "$",
+          :decimal_digits 2,
+          :rounding 0,
+          :code "UYU",
+          :name_plural "Uruguayan pesos"}],
+   [:UZS {:symbol "UZS",
+          :name "Uzbekistan Som",
+          :symbol_native "UZS",
+          :decimal_digits 0,
+          :rounding 0,
+          :code "UZS",
+          :name_plural "Uzbekistan som"}],
+   [:VEF {:symbol "Bs.F.",
+          :name "Venezuelan Bolívar",
```

Comment:
the hyperinflation in bolivar led them to get rid of the bs fuerte: it's the bs soberano now